jwst/datamodels/tests/test_schema_against_crds.py

```diff
@@ -0,0 +1,230 @@
+import os
+import logging
+from collections.abc import Iterable
+
+log = logging.getLogger('crds_test')
+log.setLevel(logging.DEBUG)
+
+os.environ["CRDS_SERVER_URL"] = 'serverless'
```

Comment:
Global environment changes affect all tests.  These changes need to be kept local to the tests that need them.

There is a pytest fixture called `jail_environ`. See `tests_models.py:74` for an example. Add the fixture to the tests and move the environment settings into the test, or another fixture to be applied to the tests.

There may be more mines-in-this-field regarding this; these variables usually need to be set before crds is imported. However, since pytest runs in a single environment, crds may have already been imported. Something to watch for.

Reply:
Asking you about the crds settings was on my to-do list - this test right now is mostly just what I used locally for the first run, but I'm working on converting it into a unit/regression test. I'm working on moving things into reasonable fixtures/tests.